### PR TITLE
Fix lower-version `dokeysto` packaging issues

### PR DESCRIPTION
**Pull request by Leonidas-from-XIV**
With extunix.0.1.1 if fails with dozens of messages like

```
gcc: error: src/eventfd.o: No such file or directory
```

### CODE REVIEW

**Comment by Leonidas-from-XIV** [comment]
Hmm, no, this looks more like an issue with the build system that extunix uses.

File: packages/conf-liblz4/conf-liblz4.1/opam [hunk 1]
```diff
@@ -11,6 +11,7 @@ depexts: [
   ["lz4-devel"] {os-distribution = "centos"}
   ["lz4-devel"] {os-distribution = "rhel"}
   ["lz4-devel"] {os-distribution = "fedora"}
+  ["lz4-devel"] {os-distribution = "oraclelinux-8"}
```
Comment (mseri) [inline packages/conf-liblz4/conf-liblz4.1/opam:14]: Does this work? I think the distribution is "ol"

**Comment by mseri** [comment]
Thanks. Can you please document what error is related to the various lower bound added in this or?

File: packages/conf-liblz4/conf-liblz4.1/opam (hunk 1 above)
Comment (Leonidas-from-XIV) [reply packages/conf-liblz4/conf-liblz4.1/opam]: It didn't. I tried `oraclelinux` and `oraclelinux-8` before. Thanks for the pointer for `ol`, I'll give that one a try.

Sidenote: how do I learn about the possible names? Especially if I am not running that distribution. I can often get package names from online search interfaces but the OS names/distribution names are difficult to figure out.

Comment (mseri) [reply packages/conf-liblz4/conf-liblz4.1/opam]: Kate added them here: https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values
I used to search among the conf-package in the opam-repository before :D

File: packages/dokeysto_camltc/dokeysto_camltc.4.0.0/opam [hunk 2]
```diff
@@ -12,8 +12,8 @@ build: [
 depends: [
   "ocaml"
   "dune" {>= "1.11"}
-  "dokeysto"
-  "camltc"
+  "dokeysto" {>= "4.0.0"}
```
Comment (Leonidas-from-XIV) [inline packages/dokeysto_camltc/dokeysto_camltc.4.0.0/opam:15]:
```
7 | module RO: functor (KV: Dokeysto.Db_gen.Key_val) -> sig
                            ^^^^^^^^^^^^^^^^^^^^^^^
Error: Unbound module Dokeysto.Db_gen
```

File: packages/dokeysto_lz4/dokeysto_lz4.4.0.0/opam [hunk 3]
```diff
@@ -12,7 +12,7 @@ build: [
 depends: [
   "ocaml"
   "dune" {>= "1.11"}
-  "dokeysto"
+  "dokeysto" {>= "4.0.0"}
```
Comment (Leonidas-from-XIV) [inline packages/dokeysto_lz4/dokeysto_lz4.4.0.0/opam:15]:
```
7 | module RO: functor (KV: Dokeysto.Db_gen.Key_val) -> sig
                            ^^^^^^^^^^^^^^^^^^^^^^^
Error: Unbound module Dokeysto.Db_gen
```

**Review by Leonidas-from-XIV — COMMENTED** [review]
I'll have to hunt down the other errors from the CI but I force-pushed so it is hard to find the older builds.

**Comment by mseri** [comment]
Thanks

**Pull request merged**